README.md

```diff
@@ -9,7 +9,7 @@
 
 `nbgitpuller` lets you distribute content in a git repository to your students
 by having them click a simple link. [Automatic
-merging](https://nbgitpuller.readthedocs.io/topic/automatic-merging.html)
```

Comment:
This link doesn't work for me and gives 404.

Reply:
Unfortunately, sphinx linkcheck only checks the links inside docs (unless README.md is included somewhere in docs).